pkg/controller/router.go

```diff
@@ -0,0 +1,31 @@
+package controller
+
+import (
+	"log"
+	"net/http"
+
+	"luis/goapi/pkg/service"
+
+	"github.com/gin-gonic/gin"
+)
+
+func Start() {
```

Comment:
Remember that function, variables, constants, structs, and interfaces that are capitalized or un uppercase are exported, this means that are visible to other packages, and so they should have a comment to begin generating part of the documentation, this is a good practice.